helm/cluster-vsphere/templates/_helpers.tpl

```diff
@@ -0,0 +1,15 @@
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "vmwarecluster.name" -}}
+{{- default .Values.nameOverride | trunc 63 | trimSuffix "-" }}
```

Comment:
I don't see `nameOverride` in `values.yaml`. And we called it `clusterName` in `cluster-openstack`.